src/App.tsx

```diff
@@ -59,7 +51,9 @@ class App extends React.Component<PropsFromRedux> {
     }
 
     if (type === "column") {
-      const newColumnOrder = Array.from(this.props.allTasks.columnOrder);
+      const newColumnOrder: string[] = Array.from(
```

Comment:
string[] here isn't necessary because the type can be inferred from other types.

Reply:
deleted string[]